ide/vscode/README.md

```diff
@@ -0,0 +1,11 @@
+# Sample configuration files for VSCode
+
+This example provides sample debug configurations that my be useful for developers using  the Microsoft(R) Visual Studio Code integrated development environment.
+
+A basic setup for Visual Studio Code is described in [Getting Started with the Raspberry Pi Pico](https://rptl.io/pico-get-started). More advanced configurations may need additional plug-ins such as *cortex-debug*.
```

Comment:
cortex-debug **is** mentioned in Chapter 7 of that PDF?

Reply:
Agreed - reference removed with [0153dafc](https://github.com/mjcross/pico-examples/commit/0153dafc773f6d3006fb85baa3f7c95a04f1484c)